configs/config.dev.yaml

```diff
@@ -57,6 +57,7 @@ vendors: []
 #  name: slack
 #  auth_token: ''
 #  base_url: 'https://slack.com/api/chat.postMessage'
+#  iris_incident_url: 'https://iris.prod.linkedin.com/incidents'
```

Comment:
Removed the internal link in commit [54f7305](https://github.com/linkedin/iris-api/pull/38/commits/54f7305b733b41c74af2821382ea571c10584f8d)